### PR TITLE
RHDEVDOCS-6033: Document the changes in the TLS termination policy for the default and user-defined Argo CD instances

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
Jira issue: [RHDEVDOCS-6033](https://issues.redhat.com/browse/RHDEVDOCS-6033) and [RHDEVDOCS-6076](https://issues.redhat.com/browse/RHDEVDOCS-6076)

Aligned team: DevTools 
Cherry-pick to versions: `gitops-docs-1.13`

Docs preview:

- [Setting up an Argo CD instance](https://77298--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance.html)
- [Installing an Argo CD instance](https://77298--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance.html#gitops-argo-cd-installation_setting-up-argocd-instance)
- [Argo CD custom resource and component properties](https://77298--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties)

SME review: @chetan-rns, @jgwest 

QE review: @varshab1210, @mehabhalodiya, @trdoyle81 (any one of them)

Peer review:  @stevsmit and @jneczypor 

Additional information: This PR also resolves [RHDEVDOCS-6076](https://issues.redhat.com/browse/RHDEVDOCS-6076).

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@chetan-rns and @jgwest:  PTAL at the PR and provide review feedback. Thanks!

**Review by chetan-rns — COMMENTED** [review]
Looks good to me @Srivaralakshmi

**Comment by Srivaralakshmi** [comment]
@varshab1210, @mehabhalodiya, @trdoyle81: PTAL at the PR and provide review feedback. Thanks!

**Comment by varshab1210** [comment]
LGTM, thanks

File: argocd_instance/setting-up-argocd-instance.adoc [hunk 1]
```diff
@@ -6,11 +6,26 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-By default, the {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions for managing certain cluster-scoped resources. To manage cluster configurations or deploy applications, you can install and deploy a new Argo CD instance. By default, any new instance has permissions to manage resources only in the namespace where it is deployed.
+By default, the {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions for managing certain cluster-scoped resources. This default Argo CD instance is also called as the default cluster-scoped instance.
```
Comment (jneczypor) [inline argocd_instance/setting-up-argocd-instance.adoc:9]:
```suggestion
By default, {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions for managing certain cluster-scoped resources. This default Argo CD instance is also called as the default cluster-scoped instance.
```
I don't think you need the article before the Gitops title.

**Review by jneczypor — COMMENTED** [review]
Just a couple of tiny suggestions. Great job! I'm still onboarding to the peer review squad, so someone will follow behind with a final review. Thanks!

**Comment by stevsmit** [comment]
Double checked Janelle's comments and think they're correct. Also, be sure to squash your commits.

File: argocd_instance/argo-cd-cr-component-properties.adoc [hunk 2]
```diff
@@ -28,4 +28,5 @@ include::modules/proc_configuring-notificationsconfiguration-crd-by-using-the-oc
 
 [role="_additional-resources"]
 == Additional resources
-* xref:../argocd_instance/argo-cd-cr-component-properties#gitops-argo-cd-notification_argo-cd-cr-component-properties[Enabling notifications with an Argo CD instance].
+* xref:../argocd_instance/setting-up-argocd-instance.adoc#gitops-argo-cd-installation_setting-up-argocd-instance[Installing a user-defined Argo CD instance]
+* xref:../argocd_instance/argo-cd-cr-component-properties#gitops-argo-cd-notification_argo-cd-cr-component-properties[Enabling notifications with an Argo CD instance]
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply argocd_instance/argo-cd-cr-component-properties.adoc:32]: Removed it. my bad, thanks!

**Pull request merged**